python/create_proxy_jwt.py

```diff
@@ -0,0 +1,131 @@
+#!/usr/bin/env python3
+#
+# Copyright (c) 2022 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+
+import argparse
+import sys
+import jwt
+import datetime
+from datetime import datetime
+from datetime import timezone
+from datetime import timedelta
+import platform
+from os import path
+import colorama
+from colorama import Fore, Back, Style
+
+def parse_args():
+    parser = argparse.ArgumentParser(description="Create JWT for Proxy (cloud-to-cloud) Requests to nRF Cloud",
```

Comment:
Maybe "Create JWT for proxy (cloud-to-cloud) requests to nRF Cloud" would be better?